Source/Core/Core/NetPlayServer.cpp

```diff
@@ -1389,16 +1400,38 @@ bool NetPlayServer::RequestStartGame()
 
   bool start_now = true;
 
-  if (m_settings.savedata_load && m_players.size() > 1)
+  if (m_settings.savedata_load)
   {
-    start_now = false;
-    m_start_pending = true;
-    if (!SyncSaveData())
+    SaveSyncInfo save_sync_info;
+    if (!CollectSaveSyncInfo(&save_sync_info))
     {
-      PanicAlertFmtT("Error synchronizing save data!");
+      PanicAlertFmtT("Error collecting save data!");
       m_start_pending = false;
       return false;
     }
+
+    if (save_sync_info.has_wii_save)
+    {
+      // Set titles for host-side loading in WiiRoot
+      std::vector<u64> titles;
+      for (const auto& pair : save_sync_info.wii_saves)
```

Comment:
Maybe `const auto& [title_id, storage]`? (assuming this doesn't produce an unused variable warning)